diff --git a/sites/all/modules/mandrill/lib/mandrill.mail.inc b/sites/all/modules/mandrill/lib/mandrill.mail.inc
index dab39a9..f7bea9b 100644
--- a/sites/all/modules/mandrill/lib/mandrill.mail.inc
+++ b/sites/all/modules/mandrill/lib/mandrill.mail.inc
@@ -254,6 +254,7 @@ class MandrillMailSystem implements MailSystemInterface {
       'text/',
       'application/pdf',
       'application/x-zip',
+      'video/',
     );
     drupal_alter('mandrill_valid_attachment_types', $valid_types);
 
