diff --git a/sites/all/modules/services/resources/node_resource.inc b/sites/all/modules/services/resources/node_resource.inc
index d7459f9..873db05 100644
--- a/sites/all/modules/services/resources/node_resource.inc
+++ b/sites/all/modules/services/resources/node_resource.inc
@@ -293,7 +293,13 @@ function _node_resource_retrieve($nid) {
  * @see drupal_form_submit()
  */
 function _node_resource_create($node) {
-  global $user;
+  $user = NULL;
+  if (isset($node['uid']) && is_numeric($node['uid'])) {
+    $user = user_load($node['uid']);
+  }
+  if (!$user) {
+    global $user;
+  }
   // Adds backwards compatability with regression fixed in #1083242
   $node = _services_arg_value($node, 'node');
   if (!isset($node['name'])) {
